src/main/kotlin/org/jetbrains/research/testspark/display/TestCasePanelFactory.kt

```diff
@@ -403,7 +402,7 @@ class TestCasePanelFactory(
             .run(object : Task.Backgroundable(project, PluginMessagesBundle.get("sendingFeedback")) {
                 override fun run(indicator: ProgressIndicator) {
                     val ijIndicator = IJProgressIndicator(indicator)
-                    if (ToolUtils.isProcessStopped(project, ijIndicator)) {
+                    if (ToolUtils.isProcessStopped(uiContext!!.errorMonitor, ijIndicator)) {
```

Comment:
Why do we also keep an instance of `errorMonitor` in the `uiContext`? Is it the same instance as in `RunnerController`?

Reply:
Because runnerController is used only in the test generation process and not in the visualization module. Hence, we are not passing it via UIContext. The only part required from this class inside UI is the `errorMonitor`.